src/lib/HAPServiceNode.ts

```diff
@@ -165,8 +166,24 @@ module.exports = (RED: NodeAPI) => {
         // Service node properties
         self.name = self.config.name
 
+        // Find a unique identifier for the current service
+        let uniqueIdentifier
```

Comment:
One more thing. We should add `uniqueIdentifier` to `self` as I see some spots where I will need that information later on.

Reply:
I just made that variable public. Are you happy with naming? A node now has an `id` and an `uniqueIdentifier`. I wonder if something like `path` would be better fitting. Also adding something like `static` to the name may make sense, because that differentiates this variable from the real `id`.